samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -58,6 +57,7 @@ import org.apache.samza.util.ScalaJavaUtil.JavaOptionals
 import org.apache.samza.util.{Util, _}
 import org.apache.samza.{SamzaContainerStatus, SamzaException}
 
+import scala.collection.JavaConverters
```

Comment:
Doesn't line 60, 61 doing the same import. Is both of them necessary?

Reply:
Good catch. I think the `JavaConverters` one without the `_` is only necessary if you directly use `JavaConverters` as a static qualifier, but that's not done in this class, so it can be removed.